### ISSUE TITLE
Tx details: Can't open delegate reg  tx details from snapshot by direct link

### ISSUE THREAD

**Issue by Efefefef** [LiskHQ/lisk-hub#1424]
### Steps to reproduce

1. Set autologin as test delegate account
1. Go to wallet
1. Scroll list to the bottom to find Delegate reg tx
1. Check it out then
1. Try to open by direct link `http://localhost:8080/#/wallet?id=2697129531259680873`

### Actual result

Wallet Activity tab
```
Uncaught (in promise) Error: Actions may not have an undefined "type" property. Have you misspelled a constant?
    at Object.performAction (<anonymous>:1:40795)
    at liftAction (<anonymous>:1:34238)
    at dispatch (<anonymous>:1:38232)
```

### Expected result

Tx details page

**Comment by osvaldovega** [comment]
I tried to reproduce this issue several times, but is not happening any more.
Could be a problem about cache or something like that what could cause the issue in first place.
I will close this issue as there is nothing to fix.

**Comment by Efefefef** [comment]
I can reproduce it. Please come, I will show to you

**Comment by osvaldovega** [comment]
This bug wont be fix on this sprint, because the design of the wallet will change and with this change the way to display and see the details of transactions delegates and other things will be have their own page, so based on that it is not necessary spend time on this task right now.

Once the new design be implemented we can review again this issue, but most likely, this problem will not exist

**Comment by Efefefef** [comment]
ok thanks @osvaldovega 
@slaweet lets put on suspended shelf?

**Comment by slaweet** [comment]
This was solved with the new design, as `/wallet?id=2697129531259680873` is no longer used. Instead clicking a row goes to the transaction detail page.
